.github/workflows/vue-ci.yml

```diff
@@ -0,0 +1,83 @@
+name: SBC-Common-Components Web CI
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
good ole master, should probably be main

Reply:
It need to find all dependency projects and change them all. Maybe leave it until Vue3 upgrade.